### PR TITLE
Added option to switch primary and secondary mouse buttons

### LINKED ISSUE #325: Option to Reverse Primary/Second Mouse Clicks
I'm using a left-handed Razer Naga on my host computer and when logging into it remotely my mouse buttons are backwards.  When on my host computer I find that BIOS has the buttons reversed as well, so it's just an issue as a result of using a left handed mouse at home.  If we had some function to reverse the primary and secondary mouse buttons within Moolight this would fix this issue.

### PR DESCRIPTION
**Pull request by annk15**
Fixes #325

### CODE REVIEW

**Comment by cgutman** [comment]
Can you add a command-line option to enable this to CommandLineParser.cpp?

Please also remove your the `.DS_Store` you added.

**Comment by annk15** [comment]
> Can you add a command-line option to enable this to CommandLineParser.cpp?
> 
> 
> 
> Please also remove the `app/.DS_Store` file you added.

Sure, I will fix that.

**Comment by annk15** [comment]
I have added a command line option to toggle between the left/right button swap.
I also did some refactoring and removed the "app/.DS_Store" file 😃

File: app/streaming/input/mouse.cpp [hunk 1]
```diff
@@ -55,6 +55,14 @@ void SdlInputHandler::handleMouseButtonEvent(SDL_MouseButtonEvent* event)
             return;
     }
 
+    if (m_SwapMouseButtons) {
+        if (button == SDL_BUTTON_RIGHT)
+            button = SDL_BUTTON_LEFT;
+        else if (button == SDL_BUTTON_LEFT)
+            button = SDL_BUTTON_RIGHT;
```
Comment (cgutman) [inline app/streaming/input/mouse.cpp:62]: These need to be using the Moonlight button definitions, not SDL's (see the switch statement above). This only works by luck (SDL's `SDL_BUTTON_LEFT` is the same value as `BUTTON_LEFT`)

Comment (annk15) [reply app/streaming/input/mouse.cpp]: I have now updated the code so it uses Moonlights button definitions.

**Comment by cgutman** [comment]
Merged, thanks!

**Pull request merged**